client/tools/mgr-push/test/test_uploadclass.py

```diff
@@ -0,0 +1,315 @@
+from unittest.mock import Mock, patch, MagicMock
+import sys
+sys.modules['platform'] = MagicMock()
+sys.modules['rpm'] = MagicMock()
```

Comment:
In fact I'm mocking the calls to the `debian` package as it is obviously used to read `.deb` files and in the tests any file is actually read. So I wonder if it would make any difference to actually import or not those packages (`debian` and `rpm`) in the unit tests.

Reply:
It was just a way to avoid the requirement of having the `python3-rpm` package installed on the testing system, as this was a problem for running the tests as a Github action (no rpm-based supported runners). I don't know is there's a better way to achieve this.

Same for the `platform` module, but this `import` is going to be removed as this is part of the traditional stack (that `platform` import occurs inside the `up2date_client` package).

Reply:
I understand why you're doing it like that, but I'm not sure that I will remember that when I open this file in 3 months from now. I think this deserves a code comment.

Reply:
All right, I misunderstood :)

Reply:
I added a comment clarifying the mocking of the `sys.modules` keys.